src/force/nep3.cu

```diff
@@ -1482,12 +1566,14 @@ static __global__ void find_descriptor(
   const double* __restrict__ g_x,
   const double* __restrict__ g_y,
   const double* __restrict__ g_z,
+  const bool is_polarizability,
```

Comment:
There is no need to modify the functions for the "temperature" version because in this case `nep_model_type=3`.